Misc/NEWS.d/next/Core_and_Builtins/2024-10-29-11-47-19.gh-issue-126119.xbAvxt.rst

```diff
@@ -0,0 +1,4 @@
+Fix a crash in DEBUG builds due to an overflow when the :attr:`co_stacksize
```

Comment:
I'd rephrase this as fixing a lack of overflow checking in `code.replace` since the bytecode compiler will never create a code object like this.